src/components/TodoApp/TodoApp.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import { client } from '../../utils/fetchClient';
+import { URL, USER_ID } from '../../utils/Url';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  setTodos: (a: Todo[] | any) => void;
```

Comment:
Avoid using 'any' in TypeScript. It helps to maintain type safety throughout your codebase. Instead, strictly type your state updates to ensure that the function only ever receives an array of Todo objects.